biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -83,29 +99,63 @@ export const InputPage = () => {
               onChange={(date: Date) => setEndDate(date)}
             />
           </div>
-          <br></br>
+        </div>
+
+        <br></br>
+
+        <div className="tooltipSBC">
+          <span className="tooltiptextSBC">SBC's are used for quick sources of energy</span>
           <div className="smallBatteryCells">
             <label>Small Battery Cells</label>
             {smallBatteryCellInput}
           </div>
+        </div>
+
+        <div className="tooltipLBC">
+          <span className="tooltiptextLBC">LBC's are used as backup energy sources to the solar panels</span>
           <div className="largeBatteryCells">
             <label>Large Battery Cells</label>
             {largeBatteryCellInput}
           </div>
-          <br></br>
+        </div>
+
+        <br></br>
```

Comment:
Prefer not to use `<br/>`, but if you do use it then know that it's a self-closing tag so you only need `<br/>` instead of `<br></br>`